bluetooth_mesh/messages/test/test_config.py

```diff
@@ -640,87 +644,97 @@
                     "corresponding_present": True,
                     "corresponding_id": 0x01,
-                    "extended_models_items": [
-                        {
-                            "model_item_index": 0x07,
-                            "element_offset": 0x00
-                        },
-                        {
-                            "model_item_index": 0x06,
-                            "element_offset": 0x00
-                        }
-                    ]
+                    "extended_models_items": {
+                        "short": [
+                            {
+                                "model_item_index": 0x07,
+                                "element_offset": 0x00
+                            },
+                            {
+                                "model_item_index": 0x06,
+                                "element_offset": 0x00
+                            }
+                        ]
+                    }
                 },
                 {
                     "extended_items_count": 0,
                     "format": ExtendedModelsItemFormat.SHORT,
                     "corresponding_present": True,
                     "corresponding_id": 0x08,
-                    "extended_models_items": []
+                    "extended_models_items": {"short": []}
                 },
                 {
                     "extended_items_count": 1,
                     "format": ExtendedModelsItemFormat.SHORT,
                     "corresponding_present": True,
                     "corresponding_id": 0x08,
-                    "extended_models_items": [
-                        {
-                            "model_item_index": 0x09,
-                            "element_offset": 0x00
-                        }
-                    ]
+                    "extended_models_items": {
+                        "short": [
+                            {
+                                "model_item_index": 0x09,
+                                "element_offset": 0x00
+                            }
+                        ]
+                    }
                 },
                 {
                     "extended_items_count": 0,
                     "format": ExtendedModelsItemFormat.SHORT,
                     "corresponding_present": False,
-                    "corresponding_id": None,
-                    "extended_models_items": []
+                    "corresponding_id": 0,
+                    "extended_models_items": {"short": []}
                 },
             ],
             "vendor_models": [
                         {
                             "extended_items_count": 0,
                             "format": ExtendedModelsItemFormat.SHORT,
                             "corresponding_present": False,
-                            "corresponding_id": None,
-                            "extended_models_items": []
+                            "corresponding_id": 0,
```

Comment:
We can't do it in different way? Right now is misleading - corresponding_id it's not 0 (which is a legal value), but it doesn't exist at all. At least we could use constant magic value NOT_EXIST?

Reply:
After a research, I think it's better to use another name of field when corresponding_id doesn't exists. Could we generate a union for this case? something like:

`union {
  corresponding_id @0 :UInt8;
  no_id @1 :Void;
}`

and this field can have a `0` value

Reply:
This change might not have a significant impact since the encoding of the value is governed by the corresponding_present field. Regardless of the value within the valid range, it will only be encoded when the corresponding_present field is set to True.

The syntax would transition as follows:

From:
```
{"extendedItemsCount": 0, "format": 0", "correspondingPresent": False, "correspondingId": 0}
```
To:

```
{"extendedItemsCount": 0, "format": 0", "correspondingPresent": True, "correspondingId": {"correspondingId": 0}}
```
Or, in the case of a None value in the construct:
```
{"extendedItemsCount": 0, "format": 0", "correspondingPresent": False, "correspondingId": {"noId": 0}}
```
IMO changing this is a waste of time. However, if you believe you have the bandwidth to implement and check it, please feel free to see if this approach works better for you.

Reply:
Ok, got it. So, I prefer to stay with single value. I'm resolving this comment.